xbmc/guilib/GUIWindowManager.cpp

```diff
@@ -771,9 +771,18 @@ void CGUIWindowManager::ActivateWindow_Internal(int iWindowID, const std::vector
   // debug
   CLog::Log(LOGDEBUG, "Activating window ID: %i", iWindowID);
 
+  // make sure we check mediasources from home
+  if (GetActiveWindow() == WINDOW_HOME)
+    g_passwordManager.strMediasourcePath = !params.empty() ? params[0] : "";
+  else
+    g_passwordManager.strMediasourcePath = "";
+
   if (!g_passwordManager.CheckMenuLock(iWindowID))
   {
-    CLog::Log(LOGERROR, "MasterCode is Wrong: Window with id %d will not be loaded! Enter a correct MasterCode!", iWindowID);
+    CLog::Log(LOGERROR,
+              "MasterCode or Mediasource-code is wrong: Window with id {} will not be loaded! "
```

Comment:
```suggestion
              "MasterCode or Mediasource-code is wrong: Window with id {} will not be loaded!"
```

Reply:
It's correct the way it is. The space is used as the string uses the next line as well

Reply:
thx for explanation @fuzzard - and it's _"just"_ the log